puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/forklift-centos-devel-pipeline.yaml

```diff
@@ -0,0 +1,15 @@
+
+- job:
+    name: forklift-centos-devel-pipeline
+    project-type: pipeline
+    triggers:
+      - reverse:
+          jobs:
+            - foreman-forklift-centos-devel-test
+          result: success
+      - timed: '0 */6 * * *'
```

Comment:
We could consider triggering this job based on the katello-nightly-rpm-pipeline

Reply:
Interesting idea, can you elaborate on what the benefits would be?

Reply:
The devel setup pulls from nightly repositories, and relies on packages there. While not the only dependency, knowing if an update to nightly broke the development environment is useful. With a timed result, pipelines could be mid-run, you get successful devel run and then nightly finishes and breaks things. Now you are 24 hours unaware from the test telling you this.

Reply:
how often does nightly run and at what interval? I was hoping to run the dev job multiple times a day

Reply:
The `katello-nightly-rpm-pipeline` is triggered whenever `foreman-nightly-rpm-pipeline` runs. The latter is ran under multiple change conditions and runs at a minimum once a day.

Reply:
sounds good, it works for me to kick the devel box off along with nightly, we can always move to a separate job if decided later